### ISSUE TITLE
Project Codename

### ISSUE THREAD

**Issue by rebeccabilbro** [DistrictDataLabs/trinket#78]
Update the project codename

**Comment by bbengfort** [comment]
There appears to be a naming conflict with: [https://trinket.io/](https://trinket.io/) which we need to sort out. Places where name has to change:

- [ ] GitHub 
- [ ] Read The Docs 
- [ ] Heroku 
- [ ] Name.com

**Comment by rebeccabilbro** [comment]
My vote is for __Cultivar__.  It's a [botanical term](https://en.wikipedia.org/wiki/Cultivar) that was originally a portmanteau of "cultivated" and "variety" - and refers to a plant whose origin or selection is primarily due to intentional human activity. I like the imagery of something that is both organic and cultivated, since this project is about the combination of statistical/ML methods and human steering. Thoughts?

**Comment by bbengfort** [comment]
Cultivar is fine with me as a code name - I think the most important thing is to document all the places where this name needs to change so that when we come up with an actual name we can do it pretty easily. 

One note, there is a [Cultivar](https://www.cultivar.com/) brand identity already; folks who sell greenhouses. I'm sure it's not going to be a conflict, but if we want to make sure we don't have to deal with it again, we can come up with something different. 

Yhat's thing is called ScienceOps, FYI, and their IDE is called Rodeo. 

I thought for sure that @rebeccabilbro was going to go for Wizard of Oz naming, and I had [Oz Symbols and Objects](http://www.bookrags.com/studyguide-wonderfulwizardoz/symbolsobjects.html#gsc.tab=0) all queued up. Thought "Cyclone" would be an interesting code name.

**Comment by rebeccabilbro** [comment]
@bbengfort I hear you on the pre-existing brand identity issue, though since the business lines are so dramatically different in this case, my sense is that we would be safe. I think the main issue with Trinket is that the product/service offerings do overlap to some extent.

That said, I really like __Cyclone__. 

@looselycoupled thoughts?

**Comment by ojedatony1616** [comment]
Just a note that there are already a couple of Python-related things named Cyclone out there, which could cause some confusion.
 
- http://cyclone.io/
- https://pypi.python.org/pypi/cyclone

**Comment by bbengfort** [comment]
@ojedatony1616  good catch; I googled "Data science cyclone" and nothing came up except for some NASA/NOAA weather related stuff. 

@rebeccabilbro I agree on the very different business safety.

**Comment by looselycoupled** [comment]
I'm good with cultivar.  I don't know if that's because it has "cult" in it or what but looks like we are in agreement.

**Comment by ojedatony1616** [comment]
I like Cultivar too. Let's go with it!

**Comment by looselycoupled** [comment]
@rebeccabilbro I see what you did there

**Comment by rebeccabilbro** [comment]
O:‑)

**Comment by looselycoupled** [comment]
@rebeccabilbro @bbengfort @ojedatony1616 

I'm making code and documentation changes in a local branch but it's tied up with another commit right now and might be another week before the changes can get merged.  How would you like to proceed for:

* readthedocs.io
* heroku.com
* name.com

I dont have account rights in either readthedocs.io or heroku.com for the Trinket project.  Is there a generic DDL account?  I have a vague memory of us agreeing to move out of Heroku and me setting up an env in AWS so perhaps we should kill the heroku account?  Did I imagine that last part?

For name.com I assume we will let the registration lapse.  Are there DDL policies over whose account owns the DNS records?

**Comment by rebeccabilbro** [comment]
@looselycoupled - I've added you as a collaborator on Heroku, though I noticed it says only @bbengfort (set up as the project owner) can change the name. As for readthedocs, do you have an account already? I tried to add you as a collaborator but couldn't find your handle.

**Comment by looselycoupled** [comment]
@rebeccabilbro Didnt realize my old readthedocs.io account was through an old email address.  I've created a new account with my gmail address.

**Comment by rebeccabilbro** [comment]
Thanks! What is your username? I don't see a @looselycoupled and can't search by email address

**Comment by looselycoupled** [comment]
Sorry, this one is "loosely.coupled" which is my preferred nick but some services don't allow a period.

I misspoke earlier about the domain registration - what I probably meant was that I don't have access to the registrar to add the DNS entry.  As AWS won't be setup for a little while, we probably just need to change from trinket.districtdatalabs.com to cultivar.districtdatalabs.com.  I've added an entry in Heroku but we need to add an entry to Name.com.

**Comment by rebeccabilbro** [comment]
@looselycoupled just added you to readthedocs!

**Comment by bbengfort** [comment]
Ok - sorry getting caught up on my email. 

- [x] The CNAME has been changed from trinket.districtdatalabs.com to cultivar.districtdatalabs.com 
- [x] The Heroku app has been changed from ddl-trinket to ddl-cultivar 
- [ ] Update git remotes for heroku 
- [ ] Update productions `ALLOWED_HOSTS` settings. 
- [ ] Make sure Allen's email address is added to `MANAGERS`/`ADMINS` settings. 

So @looselycoupled can I leave those last three up to you (well, technically all of us have to change the Git remotes).
